src/store/messagesStore.js

```diff
@@ -1416,7 +1464,11 @@ const actions = {
 	},
 
 	async removeExpiredMessages(context, { token }) {
-		context.commit('removeExpiredMessages', { token })
+		const updateRequired = context.commit('removeExpiredMessages', { token })
```

Comment:
this returns undefined
but also makes no sense, as threads are not removed from DB, and server returns them with `first: null` and `last: null`